presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -660,7 +661,7 @@ private static long fromMetastoreDistinctValuesCount(long distinctValuesCount, l
 
     public static Set<RoleGrant> fromRolePrincipalGrants(Collection<RolePrincipalGrant> grants)
     {
-        return ImmutableSet.copyOf(grants.stream().map(ThriftMetastoreUtil::fromRolePrincipalGrant).collect(toList()));
+        return ImmutableSet.copyOf(grants.stream().map(ThriftMetastoreUtil::fromRolePrincipalGrant).collect(toImmutableList()));
```

Comment:
`.collect(toImmutableSet())` would be more appropriate

Reply:
No no need to copy then. Also you can fix formatting (not to use single line chained trains).